docker/vscodium-buildpacks/entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -ex
+
+HOME="${HOME:-/home/ubuntu}"
+RENKU_MOUNT_DIR="${RENKU_MOUNT_DIR:-${HOME}/work}"
+
+mkdir -p "${RENKU_MOUNT_DIR}/.vscode/extensions"
+
+# Install python extensions
+DEFAULT_VSCODIUM_EXTENSIONS="\
+  ms-python.python \
+  ms-toolsai.jupyter \
+"
```

Comment:
This should be done in the docker image build, there's no reason to do it on launch

Reply:
Why not?

By doing this at launch:
1. Any user can customize the extensions they want to have, just by tweaking the env vars.
2. You don't need to re-build to use more extensions.

The install time is very fast, so it doesn't seem like it would annoy users. This is similar to how devcontainers work.

Reply:
Yes but those two will always be installed. So just put them in the image. And allow the user to specify additional ones if necessary.

Reply:
No, they will not alway be installed. If you specify `VSCODIUM_EXTENSIONS=" "` then no extension is installed, if you specify `VSCODIUM_EXTENSIONS="golang.go"` then you only have the go extension. The only issue at the moment is that custom env vars are not supported for Renku 2.0 sessions, so maybe this can be improved later to support command line arguments too.